src/settings/Settings.ts

```diff
@@ -2,7 +2,8 @@ export interface Settings {
 	disabledLanguages: string[];
 	customThemeFolder: string;
 	customLanguageFolder: string;
-	theme: string;
+	darkTheme: string;
```

Comment:
We need to automatically migrate the old `theme` setting to the new ones. Simply setting the current theme for dark and light, then deleting the old `theme` property from the existing settings. This is best done in the plugins `loadSettings` method.